lib/service/invoicesubscription.go

```diff
@@ -97,6 +97,7 @@ func (svc *LndhubService) ProcessInvoiceUpdate(ctx context.Context, rawInvoice *
 		svc.Logger.Errorf("Failed to commit DB transaction user_id:%v invoice_id:%v  %v", invoice.UserID, invoice.ID, err)
 		return err
 	}
+	svc.InvoicePubSub.Publish(invoice.UserID, invoice)
```

Comment:
I am wondering if this is not better done in the Invoice model on update? (or on update of special fields)